server/JsDbg.Gdb/testsuite/README

```diff
@@ -0,0 +1,5 @@
+To run these tests, install dejagnu and run "runtest" in this directory.
```

Comment:
Could you add some instructions on how to install dejagnu and point to some of its documentation?

Reply:
Done